imap_processing/cli.py

```
@@ -618,24 +626,31 @@ def do_processing(self, dependencies):
         elif self.data_level == "l1b":
             data_dict = {}
             for dependency in dependencies:
-                dataset = load_cdf(dependency, to_datetime=True)
+                dataset = load_cdf(dependency, to_datetime=True)  # type: ignore[arg-type]
+                # Todo Change
+                # Argument "to_datetime" to "load_cdf" has incompatible
```

Related to this: https://github.com/python/mypy/issues/1969

---

I am a little confused as to how to go about solving this issue.

---

We might need to change the type of `**kwargs` in the load_cdf function definition to `bool` instead of `dict`.

---

I think this is a real error, but it's also kind of annoying to fix, since the kwargs are ultimately passed into xarray_to_cdf which does not use MyPy and therefore doesn't have a defined type. I think you could avoid the issue by giving `**kwargs` in `load_cdf` a type of `object` like in this issue: `https://github.com/python/mypy/issues/6092`

---

If neither suggestion works then just leave the ignore errors statements in

---

Also, read about kwargs before attempting to understand what I'm saying here: https://realpython.com/python-kwargs-and-args/